### DISCUSSION TITLE
where can i find the latest descrpt source

### DISCUSSION THREAD

**Discussion by kleinZhf** [deepmodeling/deepmd-kit#450] [Q&A]
i am interested with your work on building the descrpt for molecules. but when i go the the op source part of your repo, there are many different suffix for a same file. so are there any comments or docs explaining what all those different suffixes mean or which version of descrpt op codes is the latest? i would like to have a deep look into your descrpt code and want to do some additional work based on it, but now are really confused with the code suffixes, looking forward to your response!

**Comment by njzjz** [comment]
> so are there any comments or docs explaining what all those different suffixes mean or which version of descrpt op codes is the latest?

Not yet. This should be an enhancement request. cc @amcadmus
